### ISSUE TITLE
Share on Hugging Face

### ISSUE THREAD

**Issue by lhoestq** [tgxs002/align_sd#2]
Hi ! Thanks for sharing this dataset :)

I was wondering if you planned to share the dataset on Hugging Face as well ? This way anyone in the research community can visualise it online and even load it in one line of python

**Comment by tgxs002** [comment]
Yes, that is on the schedule. It will probably be ready in 2 weeks.

**Comment by vishaal27** [comment]
Hey, is this available on Huggingface already?
